src/test/java/teammates/ui/webapi/UpdateFeedbackSessionActionTest.java

```diff
@@ -145,6 +145,8 @@ public void testExecute_changeDeadlineForStudents_shouldChangeDeadlinesCorrectly
                 Const.ParamsNames.NOTIFY_ABOUT_DEADLINES, String.valueOf(false),
         };
 
+        verifyNoTasksAdded();
```

Comment:
Would it be more complete to add a small test where no tasks were added when deadlines are equal?